static/main.css

```
@@ -62,11 +62,14 @@ p {
 
 #progressbar {
 	position: fixed;
-	left: 0;
+	left: -4px;
 	top: 0;
-	height: 10px;
+	height: 2px;
```

Will review later... might be better to keep it at 6px though?
```suggestion
	height: 6px;
```